environments/common/inventory/groups

```diff
@@ -13,6 +13,10 @@ login
 control
 compute
 
```

Comment:
Do we really want EESSI to be non-optional? If we want it to be optional, but deployed "by default" then it should be an empty group here, and its children set as openhpc in `environments/common/layouts/everything`

Reply:
don't know, that wasn't my intent I guess.

Reply:
ah, I see, I need a separate group to make it optional. yeah, I should do that.